samples/config_ptq_dfu2021_vgg11_without_preprocess_classification.yaml

```diff
@@ -0,0 +1,149 @@
+# 'normalize_nonZero': perform z-score normalize but with mean and std-dev calculated on only non-zero pixels
+# 'normalize_nonZero_masked': perform z-score normalize but with mean and std-dev calculated on only non-zero pixels with the stats applied on non-zero pixels
+# 'crop_external_zero_planes': crops all non-zero planes from input tensor to reduce image search space
+# 'resample: resolution: X,Y,Z': resample the voxel resolution: https://torchio.readthedocs.io/transforms/preprocessing.html?highlight=ToCanonical#torchio.transforms.Resample
+# 'resample: resolution: X': resample the voxel resolution in an isotropic manner: https://torchio.readthedocs.io/transforms/preprocessing.html?highlight=ToCanonical#torchio.transforms.Resample
+# resize the image(s) and mask (this should be greater than or equal to patch_size); resize is done ONLY when resample is not defined
+
+
+data_preprocessing:
+  {
+    # 'normalize',
+    # # 'normalize_nonZero', # this performs z-score normalization only on non-zero pixels
+    # 'resample':{
+    #   'resolution': [1,2,3]
+    # },
+    # 'resize': [128, 128], # this is generally not recommended, as it changes image properties in unexpected ways
+    # 'crop_external_zero_planes', # this will crop all zero-valued planes across all axes
+  }
+# various data augmentation techniques
+# options: affine, elastic, downsample, motion, ghosting, bias, blur, gaussianNoise, swap
+# keep/edit as needed
+# all transforms: https://torchio.readthedocs.io/transforms/transforms.html?highlight=transforms
+# 'kspace': one of motion, ghosting or spiking is picked (randomly) for augmentation
+# 'probability' subkey adds the probability of the particular augmentation getting added during training (this is always 1 for normalize and resampling)
+data_augmentation: 
+  {
+    # default_probability: 0.5,
+    # 'affine',
+    # 'elastic',
+    # 'kspace':{
+    #   'probability': 1
+    # },
+    # 'bias',
+    # 'blur': {
+    #   'std': [0, 1] # default std-dev range, for details, see https://torchio.readthedocs.io/transforms/augmentation.html?highlight=randomblur#torchio.transforms.RandomBlur
+    # },
+    # 'noise': { # for details, see https://torchio.readthedocs.io/transforms/augmentation.html?highlight=randomblur#torchio.transforms.RandomNoise
+    #   'mean': 0, # default mean
+    #   'std': [0, 1] # default std-dev range
+    # },
+    # 'anisotropic':{
+    #   'axis': [0,1],
+    #   'downsampling': [2,2.5]
+    # },
+  }
+
+## OpenVINO Post-Training Quantization (PTQ) parameters.
+# for details, see https://docs.openvino.ai/latest/pot_introduction.html
+ptq_type: "Default" # options: Default, AccuracyAware
+calib_metrics: "classification_accuracy" # options: choose one of metrics defined above. Defaults to the first metric defined in "metrics" above
+calib_ratio: 0.1 # How much of the training/validation dataset need to be used as calibration data for PTQ
```

Comment:
Why do you think it is more convenient to define the size of the calibration dataset as a ratio with respect to training/validation dataset?

Reply:
This is the usual approach I take when I work with customer models. Do you have suggestion on the common practice w.r.t this?

Reply:
The absolute number of samples is more meaningful to me. This is what we have in the NNCF tutorials as well. To tell the truth, when it comes to the public dataset, I don't know how many samples are in the dataset unless I check it somehow (usually in the code which is not trivial sometimes).
BTW, NNCF can handle the situation when the user specifies 300 samples for calibration but only 200 are available. In this case, NNCF will take only 200 without failure.

Reply:
Updated the script to allow both ratio and absolute number of sample size